### PR TITLE
elixir: new recipe

### PR DESCRIPTION
**Pull request by return**
Elixir is a functional programming language built on top of the Erlang VM and is used for building scalable applications. 

Elixir fails to build from source due to a segfault in the Erlang VM. Until this is patched, the pre-compiled version was tested instead and so far works perfectly on x86_64.

### CODE REVIEW

File: dev-lang/elixir/elixir-1.4.2.recipe [hunk 1]
```diff
@@ -0,0 +1,45 @@
+SUMMARY="A dynamic, functional language for building scalable applications"
+DESCRIPTION="Elixir leverages the Erlang VM, known for running \
+low-latency, distributed and fault-tolerant systems, while also \
+being successfully used in web development and the embedded software \
+domain."
+HOMEPAGE="http://elixir-lang.org"
+COPYRIGHT="2012-2017 Plataformatec"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/elixir-lang/elixir/releases/download/v$portVersion/Precompiled.zip"
```
Comment (korli) [inline dev-lang/elixir/elixir-1.4.2.recipe]: The source file should be renamed with SOURCE_FILENAME.

File: dev-lang/elixir/elixir-1.4.2.recipe [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	cmd:iex = $portVersion
+	cmd:mix = $portVersion
+	"
+REQUIRES="
+	haiku
+	cmd:erl >= 18.0
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
```
Comment (korli) [inline dev-lang/elixir/elixir-1.4.2.recipe]: I think you don't need haiku_devel.

**Comment by korli** [comment]
Such precompiled recipes are usually suffixes with _bin to clearly indicate that nothing is built by the recipe.

**Comment by miqlas** [comment]
Wouldn't it be better to re-port this stuff?

**Comment by Begasus** [comment]
@return any progress on this?

**Pull request merged**